index.html

```diff
@@ -0,0 +1,211 @@
+    <!------------------------------MAIN  END------------------------------------->
+
+    <!------------------------------PROGRAMS STARTS------------------------------->
+    <section class="events">
+      <div class="holder">
+        <h2>Main Program</h2>
+        <hr />
+        <div class="events-content">
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-analysis"></i>
+              <h3>Exhibition</h3>
+            </div>
+            <p>
+              40+ Pharmaceutical & FMCG companies have confirmed their
+              participation as exhibitors at the conference.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-tv-retro"></i>
+              <h3>Excursion</h3>
+            </div>
+            <p>
+              Explore the vast Historic collections at the national Museum, Walk
+              through a range of art works at the Nike Art Garlary
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-comment-alt-notes"></i>
+              <h3>Workshop</h3>
+            </div>
+            <p>
+              The mandatory continuing professional development program that
+              will be hosted in this year’s conference is focused on Vaccination
+              Training for Pharmacists.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-presentation-edit"></i>
+              <h3>Novelty Match</h3>
+            </div>
+            <p>
+              There will be a Novelty Match between the different state arm of
+              the association.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-palette"></i>
+              <h3>Group Dynamics</h3>
+            </div>
+            <p>
+              Every state arm of the association will be required to exhit or
+              showcase their progress and activites within the past one year
+            </p>
+          </div>
+          <div class="join">
+            <a href="#">Join ACPN 2023.</a>
+          </div>
+        </div>
+        <p class="show">SEE THE WHOLE PROGRAM</p>
+      </div>
+    </section>
+
+    <!------------------------------PROGRAMS  END--------------------------------->
+
+    <!------------------------------SPEAKERS  STARTS------------------------------>
+    <section class="speakers">
+      <div class="holder">
+        <h2>Featured Speakers</h2>
+        <hr />
+        <div class="guest-speakers"></div>
+        <!-- <div class="speakers-detail">
+          <img src="" alt="" />
+          <div class="details">
+            <h4>Folashade Lawal</h4>
+            <p class="title">Pharmacist</p>
+            <p class="position">Head Of Pharmacy</p>
+          </div>
+        </div> -->
```

Comment:
- To help improve the readability and maintenance of your code, It's not advisable to leave commented out code in your codebase. Please fix this 🙏